recipe/migrations/jpeg_to_libjpeg_turbo.yaml

```diff
@@ -7,7 +7,7 @@ __migrator:
   paused: false
 
 libjpeg_turbo:
-  - 2.1.5
+  - 3
```

Comment:
Why update the existing migrator? That means migrated feedstocks will just pick up libjpeg 3 upon the next rerender - do we trust the API stability of libjpeg enough that this cannot reasonably break?

Reply:
hmm ok i didn't understand this detail. sorry lets revert

Reply:
i mostly don''t understand the interplay of the two migrations. I didn't have a chance to study whether or not we can fully close this migration.